shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/core/presentation/BirtdayDialog.kt

```diff
@@ -0,0 +1,36 @@
+package com.vml.tutorial.plantshop.core.presentation
+
+import androidx.compose.material3.DatePicker
+import androidx.compose.material3.DatePickerDialog
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.rememberDatePickerState
+import androidx.compose.runtime.Composable
+import com.vml.tutorial.plantshop.MR
+import com.vml.tutorial.plantshop.core.utils.exts.orZero
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BirthdayDialog(onDateConfirm: (Long) -> Unit, onDismissRequest: () -> Unit) {
```

Comment:
Can you set a default start date e.g: (now - 18 years)? It just an idea so you can ignore this :)

Reply:
I like the idea. Default start date added